### PR TITLE
Don't expose 2 script properties in Inspector

### PR DESCRIPTION
**Pull request by volzhs**
Screenshot from 2019-12-20 13-48-33

### CODE REVIEW

**Comment by aaronfranke** [comment]
@volzhs Is this still desired? If so, it needs to be rebased on the latest master branch. While there are no conflicts, rebasing is important so that reviewers can easily test the PR.

File: editor/editor_inspector.cpp [hunk 1]
```diff
@@ -1475,6 +1475,7 @@ void EditorInspector::update_tree() {
 		_parse_added_editors(main_vbox, ped);
 	}
 
+	bool has_scripts = false; // for checking if exposed "Script/scripts" by MultiNodeEdit.
```
Comment (KoBeWi) [inline editor/editor_inspector.cpp:1478]:
```suggestion
	bool has_scripts = false; // For checking if exposed "Script/scripts" by MultiNodeEdit.
```

**Comment by KoBeWi** [comment]
I'm not sure if this is a correct fix. Instead of adding another inspector hack, this should be changed in MultiNodeEdit. But I tried a few things and nothing seems to help, so maybe it's the only way.

**Comment by raulsntos** [comment]
This is most likely caused because, as described in https://github.com/godotengine/godot/issues/51077#issuecomment-984028025, `Object::get_property_list` always adds the `script` property so when `MultiNodeEdit` implements the virtual `_get_property_list` method, even though it does not add a `script` property it will still be added. That's why no change in `MultiNodeEdit` can prevent the `script` property being added.

In #55530 I fixed the inspector showing the `script` twice by making `EditorDebuggerRemoteObject` not include the `script` property (since it will be added by `Object` anyway) but this issue is different, it's not about preventing a property showing twice. In this case we don't want the `script` property showing at all because it's supposed to be replaced with the `scripts` property (notice the plural form).

So with that said, the only way I can see to fix this would be to make the change in the inspector (as this PR does) or change the way `Object::get_property_list` works to not add the `script` property itself and let implementations of `_get_property_list` add it themselves which would be a much bigger change than this PR but might be worth considering for the long term.

**Comment by akien-mga** [comment]
Superseded by #57654.

**Pull request closed without merging**